scripts/release_scripts/deploy.py

```diff
@@ -220,12 +221,17 @@ def update_and_check_indexes(app_name):
         'https://console.cloud.google.com/datastore/indexes'
         '?project=%s') % app_name
     gcloud_adapter.update_indexes(INDEX_YAML_PATH, app_name)
-    if not gcloud_adapter.check_all_indexes_are_serving(app_name):
-        common.open_new_tab_in_browser_if_possible(indexes_page_url)
-        raise Exception(
-            'Please wait for all indexes to serve, then run this '
-            'script again to complete the deployment. For details, '
-            'visit the indexes page. Exiting.')
+
+    if gcloud_adapter.check_all_indexes_are_serving(app_name):
+        return
```

Comment:
Maybe remove lines 225, 226, and 231 and just have lines 233 & 234. Wdyt?

Reply:
Done.